logic/cacheLogic.js

```diff
@@ -134,56 +134,61 @@ module.exports = class CacheLogic {
   }
 
   static async getAllTips(blacklist = true) {
-    const [allTips, tipsPreview, chainNames, commentCounts, blacklistedIds, localTips] = await Promise.all([
-      CacheLogic.getTips(), LinkPreviewLogic.fetchAllLinkPreviews(), CacheLogic.fetchChainNames(),
-      CommentLogic.fetchCommentCountForTips(), BlacklistLogic.getBlacklistedIds(), TipLogic.fetchAllLocalTips(),
-    ]);
-
-    let tips = allTips;
-
-    // filter by blacklisted from backend
-    if (blacklist && blacklistedIds) {
-      tips = tips.filter(tip => !blacklistedIds.includes(tip.id));
-    }
+    const keys = ['CacheLogic.getAllTips'].concat(blacklist ? ['blacklisted'] : ['all']);
```

Comment:
why not `['CacheLogic.getAllTips', blacklist]` to have true, false as cache keys?

Reply:
This way the cache keys are more descriptive and I was not sure how booleans were handled in the cache key.